lib/core/xskdev/xskdev.c

```diff
@@ -974,6 +974,17 @@ xskdev_socket_create(struct lport_cfg *c)
     if (fq_reserved(xi, umem->fq_size) < 0)
         CNE_ERR_GOTO(err, "Failed reserved fill of FQ\n");
 
+    do {
+        socklen_t optlen = sizeof(struct xdp_statistics);
+        int ret, fd;
+
+        fd  = xsk_socket__fd(xi->rxq.xsk);
+        ret = getsockopt(fd, SOL_XDP, XDP_STATISTICS, &xi->orig_stats, &optlen);
```

Comment:
Are you sure we need to get the stats when we create a socket? I would think any new socket would already have zeroed  stats, right?

Reply:
Most likely, I was thinking about how I did it in pktgen and in that case I did need to grab the stats from the NIC.

I will remove this one and assume the stats are zero.

Reply:
Removed the code and assume the XDP stats are zero.